src/index.html

```diff
@@ -7,8 +7,27 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav>
+        <ul>
+          <a href="#" class="is-active">Apple</a>
```

Comment:
```suggestion
        <ul>
         <li><a href="#" class="is-active">Apple</a></li>
```
you forgot to cover each `a` tag with `li` in `ul`